docs/ref/models/lookups.txt

```diff
@@ -57,6 +57,10 @@ register lookups on itself. The two prominent examples are
         and checks if any has a registered lookup named ``lookup_name``, returning
         the first match.
 
+    .. method:: get_lookups()
+
+        Returns the set of all :class:`Lookup` classes registered in the class.
```

Comment:
Isn't it a dictionary of lookup names to Lookup classes?
```
>>> Question._meta.fields[0].get_lookups()
{'contains': django.db.models.lookups.Contains,
 'endswith': django.db.models.lookups.EndsWith,
 'exact': django.db.models.lookups.Exact,
 'gt': django.db.models.lookups.GreaterThan,
...
```

Reply:
Yes. Brain freeze up I think.